packages/core/src/App/Components/Elements/Notifications/empty-notification.jsx

```diff
@@ -8,9 +8,9 @@ const EmptyNotification = () => (
             <Icon icon='IcBox' className='notifications-empty__icon' size={64} color='secondary' />
             <div className='notifications-empty__content'>
                 <h4 className='notifications-empty__header'>{localize('No notifications')}</h4>
-                <span className='notifications-empty__message'>
+                <Text size='xxs' color='less-prominent'>
```

Comment:
Same here, you should pass in the `line_height` as the default `xs` is incorrect, it should be `m`. Perhaps we should make basic props such as `size` and `line_height` required props?

Reply:
Actually, we have changed the default, and now the default is m as you can see in the text.js